### ISSUE TITLE
irrelevant named tensor warning

### ISSUE THREAD

**Issue by seherozcelik** [pytorch/pytorch#60342]
## 🐛 Bug

warning:

/usr/local/lib/python3.7/dist-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)

## To Reproduce

for this training

def train():
  model_conv = models.alexnet(pretrained=True)
  model_conv.classifier.add_module('7', nn.ReLU(inplace=True))
  model_conv.classifier.add_module('8',nn.Linear(in_features=1000, out_features=1, bias=True))
  model_conv = model_conv.cuda()

  loss_func = nn.MSELoss()
  optimizer_conv = torch.optim.Adadelta(model_conv.parameters(), lr=0.001, rho=0.9, eps=1e-06)
  #lr_schlr = lr_scheduler.StepLR(optimizer_conv,step_size=10,gamma=0.7)

  train_loader = data.DataLoader(getData('tr_calib'), batch_size=2, shuffle=True)
  val_loader = data.DataLoader(getData('val_calib'), batch_size=2)
  tst_loader = data.DataLoader(getData('tst_calib'), batch_size=2)

  losses = []
  val_losses = []
  for epoch in range(30):
    loss_sum = 0
    model_conv.train()
    for inputs, labels in train_loader:
      inputs = torch.FloatTensor(inputs).cuda()
      outputs = model_conv(inputs)
      labels = torch.unsqueeze(labels.cuda(),1)
      
      optimizer_conv.zero_grad()

      loss = loss_func(outputs, labels)
      loss_sum += loss.item()
      loss.backward()
      optimizer_conv.step()

    losses.append(loss_sum / len(train_loader))
    _, _, val_loss = model_eval(model_conv, val_loader);
    val_losses.append(val_loss)
    #lr_schlr.step()
    #print(optimizer_conv.param_groups[0]['lr'])
  
  tr_lbls, tr_prds, _ = model_eval(model_conv, train_loader);
  tst_lbls, tst_prds, _ = model_eval(model_conv, tst_loader);

  return losses, val_losses, tr_lbls, tr_prds, tst_lbls, tst_prds

## Expected behavior

no warning

## Environment

default google colab

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by ngimel** [comment]
Dupe of #60053, fixed on master

**Comment by seherozcelik** [comment]
Hi,
I am using 1.9.0+cu102, and still taking the warning. Thanks... Seher...
